aadl.library/Timing_Properties.sysml

```diff
@@ -1,4 +1,14 @@
+/* SysML v2 Domain Library for AADL:
```

Comment:
We should also have a conversation about style. The changes to this file are style-only: file comments, import statement, and whitespace. What should the style be for the library?

Reply:
We started this discussion during the Reston meeting.
Maybe the best would be to align with other existing sysml standard libraries ?